gateway/1.5/modules/ROOT/pages/policies-pdk-troubleshooting.adoc

```diff
@@ -0,0 +1,58 @@
+= Policy Development Kit Troubleshooting
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+When using the Policy Development Kit (PDK), you might encounter issues with PDK itself. See the following sections to help you resolve these issues:
```

Comment:
The intro implies multiple sections, but only one bullet appears.

Reply:
I'm kind of assuming that there will be more sections eventually, but I could remove this for now?